### PR TITLE
If

### PR DESCRIPTION
**Pull request by Kartynnikk**

### CODE REVIEW

File: if.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>If homework</title>
+</head>
+
+<body>
+    <script>
+        let nameJavaScript = prompt("Какое официальное название JavaScript ?");
```
Comment (kathrynowy) [inline if.html:13]: move everything into index.js

File: if.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        if (nameJavaScript == "ECMAScript") {
```
Comment (kathrynowy) [inline if.html:14]: use strict comparison

File: if.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+            alert('True');
+        } else {
+            alert('Не знаете? ECMAScript!');
+        }
+    </script>
+
+    <script>
+        let number = prompt("Введите число", 0);
+        number = Number(number);
```
[the remark this answers is not in the document]
Comment (kathrynowy) [reply if.html:23]: check this functionality. if the user enters 'abc'

File: if.html [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+            alert(1);
+        } else if (number < 0) {
+            alert(-1);
+        } else {
+            alert(0);
+        }
+    </script>
+
+    <script>
+        let sum = (a + и < 4) ? "Мало" : "Много";
```
Comment (kathrynowy) [inline if.html:34]: a + и?

File: if.html [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    </script>
+
+    <script>
+        let sum = (login == 'Сотрудник') ? 'Привет' :
```
Comment (kathrynowy) [inline if.html:38]: same use strict comparison

File: if.html [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+        } else {
+            alert("False")
+        }
+    </script>
+
+    <script>
+        let min = promt(" Введите число", 15);
+        if (min >= 0 && min <= 15) {
+            alert(" Первая часть ");
+        } else if (min >= 15 && min <= 30) {
```
Comment (kathrynowy) [inline if.html:57]: and here the same it's ok for 15

File: if.html [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+        } else if (min > 30 && min <= 45) {
+            alert(" Третья часть ");
+        } else if (min >= 45 && min <= 60) {
+            alert(" Четевертая часть ");
+        }
+    </script>
+
+    <script>
+        let a = promt(" ");
+        let result = (a == 0) ? " Верно " : " Неверено ";
```
[the remark this answers is not in the document]
Comment (kathrynowy) [reply if.html:68]: do you need to write  == 0?
can you use another condition


what boolean value will be if a variable has value 0?

File: if.html (hunk 1 above)
Comment (Kartynnikk) [reply if.html]: fixed

File: if.html (hunk 2 above)
Comment (Kartynnikk) [reply if.html]: fixed

File: if.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (Kartynnikk) [reply if.html]: fixed

File: if.html (hunk 5 above)
Comment (Kartynnikk) [reply if.html]: fixed

File: if.html [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+    <script>
+        let sum = (login == 'Сотрудник') ? 'Привет' :
+            (login == 'Директор') ? 'Здравствуйте' :
+            (login == '') ? 'Нет логина' :
+            '';
+    </script>
+
+    <script>
+        let number = prompt(" Чему равна переменная а? ", 0);
+        if (number == 10) {
```
[the remark this answers is not in the document]
Comment (Kartynnikk) [reply if.html]: fixed

File: if.html [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+            alert("True");
+        } else {
+            alert("False")
+        }
+    </script>
+
+    <script>
+        let min = promt(" Введите число", 15);
+        if (min >= 0 && min <= 15) {
```
[the remark this answers is not in the document]
Comment (Kartynnikk) [reply if.html]: fixed

File: if.html (hunk 7 above)
[the remark this answers is not in the document]
Comment (Kartynnikk) [reply if.html]: fixed

File: index.html [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+        alert(userAge);
+    </script>
+
+    <script>
+        let message = confirm(" Are you my boss ?");
+        alert(message);
+    </script>
+
+    <script>
+        let a = prompt("Первое число?", 1),
```
[the remark this answers is not in the document]
Comment (Kartynnikk) [reply index.html:47]: fixed

**Review by kathrynowy — APPROVED** [review]

**Pull request merged**